Typo removed

m8tin opened the pull request:
I read your interesting article and hope to contribute with the small change.

spxak1 commented:
Thanks. I have corrected it but did not merge, since my mistake was on what is a forward and what a back slash. So I switched the words rather than the slashes. Thanks for the correction.

m8tin commented:
> Thanks. I have corrected it but did not merge, since my mistake was on what is a forward and what a back slash. So I switched the words rather than the slashes. Thanks for the correction.

Hey, now I am a little confused. :-)
I looked this up: https://en.wikipedia.org/wiki/Backslash

Is it possible you switched the words of my corrected version instead of the faulty one?
I looked to my commit and now I am also not safisfied with it.

In my opinion the sentence should be:
The initrd file is in the same location as the kernel. But notice how the path is written with backward slashes (\\) rather than forward slashes (/) as expected.

spxak1 commented:
Sorry, I will check again. I might have messed up. Thanks!!
